src/deserialize_sequence.cc

```diff
@@ -335,6 +335,75 @@ Step deserialize_step(const std::filesystem::path& path)
     return step;
 }
 
+std::istream& operator>>(std::istream& stream, Sequence& seq)
+{
+    std::string line;
+    std::string step_setup_script{};
+    bool has_step_setup_script = false;
+    bool load_step_setup_script = false;
+
+    while(std::getline(stream, line, '\n'))
+    {
+        auto tmp = gul14::trim_sv(line);
+        if (not load_step_setup_script)
+        {
+            if (not has_step_setup_script and tmp.find_first_of("-- step setup: [[[[[") != std::string::npos)
+            {
+                tmp.remove_prefix(std::string("-- step setup [[[[[").size());
```

Comment:
I think you have a better approach than using `std::string::remove_prefix(..::size_type)`. The only solution I found out is the complex stuff above which can be solved much smarter. Moreover I suppose it is better to put the constant string as constant. And maybe you have another idea against surrounding the `-- step setup` with five square brackets (why do I use here a Lua comment :-/ ... must also be removed).

Reply:
Maybe it is obsolete if we decide to have a more dedicated Lua sequence Lua script. See comment [here](https://github.com/taskolib/taskolib/pull/49#discussion_r1039231481).

Reply:
Yes, I would simplify this. See below.